_drafts/release-osiam-3.0.md

```diff
@@ -3,81 +3,124 @@ title:  "Release: OSIAM 3.0"
+The only artifact that we distribute is the OSIAM `.war` file.
+This file already contains everything OSIAM needs, e.g. templates and web assets.
+These files will be automatically extracted in a new "home directory", when OSIAM starts.
+But you can still extract them manually if you like.
+The home directory can be placed anywhere you like, but defaults to `~/.osiam`.
+It is the place where everything OSIAM needs at runtime lives now.
+This also includes the configuration file, making it a replacement for `/etc/osiam` in a default 2.x installation.
+
+Speaking of configuration file: We changed the supported format for it from a Java properties file to YAML.
+This means you have to rewrite your current configuration files, making it a single YAML file.
+Fortunately, it's very easy to do so.
+See the [migration notes](https://github.com/osiam/osiam/blob/master/docs/migration.md#create-a-new-configuration-file)
+for more details.
+
+Another great feature of OSIAM 3.0 is, that you don't need a Tomcat server anymore.
+While deploying OSIAM to Tomcat is still possible, you can also run it as a standalone application now.
+Just start OSIAM with `java -jar osiam.war`.
+But that's not all.
+The `.war` file is not only an executable Java web archive, it's actually a fully executable file.
+So, if you set the executable permission on it, you can start it like any other application on the command line:
+
+```sh
+chmod +x osiam.war
+./osiam.war
+```
+
+Having this feature made it possible to add a simple command line interface to OSIAM.
+With OSIAM 3.0 you can execute some basic commands directly on the command line.
+Currently there two commands defined:
+
+Creating and initializing the home directory:
+
+```sh
+./osiam.war initHome [--osiam.home=/path/to/home/directory]
+```
+
+And creating or updating the database schema:
+
+```sh
+./osiam.war migrateDb [--osiam.home=/path/to/home/directory]
+```
+
+Finally, custom SCIM 2.0 extensions can now be easily configured in the configuration file.
+This is the first feature based on the new YAML format for configuration.
+Now you don't have to directly change data in the data base anymore.
+You can see how it works in the [installation guide](https://github.com/osiam/osiam/blob/master/docs/detailed-reference-installation.md#configuring-scim-extension).
 
-TODO: Write me
-
-### Running Standalone
-
-TODO: Write me
-
-### Home Directory
-
-TODO: Write me
-
-### Command Line Interface
+## Security
 
-TODO: Write me
+We replaced the custom algorithm for hashing passwords with [bcrypt](https://en.wikipedia.org/wiki/Bcrypt).
+This is more secure, because bcrypt is mature and widespread hashing algorithm.
+It's also future-proof, because you can increase the cost factor, which increases the time to validate a hash.
+The default cost factor is currently set to `13`, which is fair enough these days.
+For now, there is no way to change the cost factor, but this will come in a later version of OSIAM.
```

Comment:
For now, there is no way to change the cost factor, but this will be provided in a later version of OSIAM.